packages/aleph-holders/src/types.ts

```diff
@@ -48,3 +48,20 @@ export type BalanceQueryArgs = {
   skip?: number
   reverse?: boolean
 }
+
+export type SnapshotQueryArgs = {
+  timestamp?: number
+}
+
+export type SolanaBalance = {
+  account: string
+  owner: string
```

Comment:
Same here, the `owner` is more similar to the `account` field of the ERC20Balance